content/2018.02.using-spread-operator-with-webpack.rst

```diff
@@ -0,0 +1,86 @@
+How to use the JavaScript spread operator with Webpack
+######################################################
+
+:lang: en
+:date: 2018-02-01
+:url: spread-operator-with-webpack
+:summary: A trick to configure webpack so that it handles the spread operator.
+
+
+What is the spread operator?
+============================
+
+One of the cool feature of JavaScript is the spread operator feature:
```

Comment:
Object rest spread https://github.com/tc39/proposal-object-rest-spread

It's only *stage 3 proposal*. 

Link to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator ?

Reply:
> It's only stage 3 proposal.

What does it means?